libs/datamodel/core/src/transform/ast_to_dml/validate.rs

```diff
@@ -30,15 +30,11 @@ impl<'a> Validator<'a> {
         Self { source }
```

Comment:
Could we have the database in here already? It feels weird to have only the source in the constructor...

Or then just get rid of the whole `Validator` struct and instead just use functions.

Reply:
Yep, I just did the minimal thing there, but I think we should do one of these two things, at some point.

Reply:
I know :) And I'm just wondering here what is the purpose of `Validator` as a struct.